model/src/main/kotlin/config/RuleViolationResolutionReason.kt

```diff
@@ -20,6 +20,11 @@ package com.here.ort.model.config
  */
 
 enum class RuleViolationResolutionReason {
+    /**
+     * The rule violation cannot be fixed an is acceptable in this case.
+     */
+    CANT_FIX_EXCEPTION,
```

Comment:
I'd prefer to more formal `CANNOT_FIX_EXCEPTION`.

Reply:
I'd be ok with that. Are you aware of the `ErrorResolutionReason.CANT_FIX_ISSUE` ?

Reply:
Damn. Then let's stick to `CANT` for consistency.